internal/iam/command_user_invitation_list.go

```diff
@@ -34,11 +34,6 @@ func (c invitationCommand) listInvitations(cmd *cobra.Command, _ []string) error
 		return err
 	}
 
-	if len(invitations) == 0 {
```

Comment:
so it will print nothing if no invitations were found? Or it should be a `None found`

Reply:
It'll print "None found." instead of "No invitations found." This is what all of our other commands say!